### DISCUSSION TITLE
Wiki updates needed

### DISCUSSION THREAD

**Discussion by mcuee** [avrdudes/avrdude#1157] [General]
With the merge of PR #1132 and  PR #1146, we need to update the Wiki Linux build instruction to include libreadline dependancy.

As of now I do not have the rights to edit Wiki.

**Comment by mcuee** [comment]
@dl8dtl 
Just wondering if you can grant me the Wiki edit rights. Thanks.

**Reply by mcuee** [reply]
@dl8dtl and @mariusgreuel 
Ping. Thanks.

#1156 has been merged as well, for macOS building instruction may need to be updated as well.

**Reply by mcuee** [reply]
Thanks, Now I have the right to update Wiki.

**Comment by dl8dtl** [comment]
Great!

**Comment by mcuee** [comment]
First update is for Linux build instruction -- I add `libreadline-dev` to Debian based system.
https://github.com/avrdudes/avrdude/wiki/Building-AVRDUDE-for-Linux

**Comment by mcuee** [comment]
Second update is for macOS build instruction. I have include readline in the dependancies and added the instruction for macOS Apple Silicon.
https://github.com/avrdudes/avrdude/wiki/Building-AVRDUDE-for-macOS

@dl8dtl 
Please review the changes for MacPorts. I believe it is correct.

@MCUdude 
Please review the changes for HomeBrew. 

The system install instruction is not correct for Homebrew as it does not point to the right install directory, so I only mention it for MacPorts. But I will try to update this later when I figured out the right instruction.

**Reply by mcuee** [reply]
Wiki updated again to remove the dependency to GNU Readline.

**Comment by mcuee** [comment]
Updated the FreeBSD build instructions as well -- I add `readline` to the dependancies.
https://github.com/avrdudes/avrdude/wiki/Building-AVRDUDE-for-FreeBSD

**Comment by mcuee** [comment]
I have also included mingw32/mingw64 nightly builds here.
https://github.com/avrdudes/avrdude/wiki/Getting-Nightly-Builds-for-AVRDUDE

This is because of Issue #968.
* #968

**Comment by mcuee** [comment]
@mariusgreuel 

I have done some minor updates on Windows related pages. Please take a look as well when you got some time.

https://github.com/avrdudes/avrdude/wiki/The-story-on-libusb-for-Windows
https://github.com/avrdudes/avrdude/wiki/Building-AVRDUDE-for-Windows-using-Visual-Studio
https://github.com/avrdudes/avrdude/wiki/Building-AVRDUDE-for-Windows-using-MSYS2
https://github.com/avrdudes/avrdude/wiki/Building-AVRDUDE-for-Windows-using-Cygwin

**Comment by mariusgreuel** [comment]
@mcuee Your changes are just fine. I tend to use stronger words because I think we need to get everybody to switch to WinUSB, but I agree that softer wording is more appropriate for this wiki.
